lib/api/hub_config_api.rb

```diff
@@ -35,15 +32,25 @@ def signing_certificates(entity_id)
 
 private
 
-  def encryption_cert_path(entity_id)
-    URI.join(HUB_HOST, CERTIFICATES_ROUTE, CERTIFICATE_ENCRYPTION_ENDPOINT % { entity_id: CGI.escape(entity_id) }).to_s
+  def use_secure_header(environment)
+    hub_environment(environment, 'secure-header') == 'true' && environment == 'integration'
```

Comment:
let's not hardcode these conditions
```suggestion
    hub_environment(environment, 'secure-header') == 'true'
```

Reply:
Ok, I thought this applied only to integration